src/models/VendingMachine.js

```diff
@@ -59,4 +69,36 @@ export class VendingMachine {
       return coinCount;
     });
   }
+
+  returnTheRestAmountsByCoins(coinInputControllerState) {
+    const restAmount = coinInputControllerState.flushTotalAmount();
+
+    const res = this.#divideNumberInDivideLevels(restAmount, COINS);
+    this.#setVendingMachineInLocalStorage();
+
+    return res;
```

Comment:
COINS는 인자로 주입해주지 않아도 될 것 같고 네이밍도 조금 바꿔보면 좋을 것 같아요
`divideNumberInDivideLevels`가 상세하지만 어떤 동작을 하는지 유추하기 힘들 것 같아요
```suggestion
    const coins = this.#exchangeCoins(restAmount);
    this.#setVendingMachineInLocalStorage();

    return coins;
```